script/upload

```diff
@@ -14,7 +14,7 @@ fi
 
 function generate::attachments
 {
-    ./build-flannel-resources.sh
+    ./script/build-resources
```

Comment:
This rename will [break our build & release process](https://github.com/charmed-kubernetes/jenkins/blob/master/jobs/includes/charm-support-matrix.inc#L37).  It's an easy enough fix on the CI side, but we'd also have to backport this change to stable for patch releases as well.

Reply:
What would you suggest? I only saw a stand-alone script, and there was a folder for all the scripts => I moved it.
I think moving the script is reasonable, but at the same time I understand that it will break a few things. What would you say if I added a symlink?

Reply:
Ah, it looks like the `script` dir was part of a short-lived attempt to standardize the build process around convention rather than a reference, but that didn't work out and was [removed from some charms](https://github.com/charmed-kubernetes/charm-kubernetes-master/pull/145/files#diff-99b83631df533ea1d9c97396a043a4670464f523d0c1e60b0a1cab6dddd440f9) but apparently some others got missed.  Would you mind reverting this rename and I'll remove the `script` dir in another PR?

Reply:
Ok, I reverted this rename. Please wait for the [tests](https://github.com/r-f-g/charm-flannel/runs/2898210624?check_suite_focus=true).

Reply:
Created PR #72 to clean up those files.